lib/Basic/FileManager.cpp

```diff
@@ -295,11 +279,13 @@ const FileEntry *FileManager::getFile(StringRef Filename, bool openFile,
   UFE.Dir     = DirInfo;
   UFE.UID     = NextFileUID++;
   UFE.UniqueID = Status.getUniqueID();
-  UFE.IsNamedPipe = Status.getType() == llvm::sys::fs::file_type::fifo_file;
```

Comment:
I think you need to keep this line, though I haven't tested it. Last time it was changed in [a different commit](https://github.com/apple/swift-clang/commit/e6138c2236eeb616c2f2b9774be0640fb0e026db#diff-0025af005307891b5429b6a834823d5eR295), not the one you are reverting.

Reply:
Oops. Bad merge resolution, thank you for catching this!